packages/@sanity/form-builder/src/sanity/inputs/SanityImageInput.tsx

```diff
@@ -38,7 +37,6 @@ export default React.forwardRef(function SanityImageInput(props: Props, forwarde
   return (
     <ImageInputWithValuePath
       {...props}
-      resolveUploader={resolveUploader}
```

Comment:
why was this removed?

Reply:
I removed it because, at least from the imageInput file, the linting was saying that the prop itself was never being used. I don't recall specifically why, but we started using the resolverUploader directly from `sanity/uploads/resolveUploader` 🤔 I felt it didn't make sense to keep it since that was the case